themes/bootstrap3/theme.config.php

```diff
@@ -61,5 +61,42 @@
             'layoutClass' => 'VuFind\View\Helper\Bootstrap3\LayoutClass',
             'search' => 'VuFind\View\Helper\Bootstrap3\Search'
         ]
+    ],
+    'icons' => [
+        'defaultSet' => 'FontAwesome',
+        'sets' => [
+            /**
+             * Define icon sets here.
+             *
+             * All sets need:
+             * - 'template': which template the icon renders with
+             * - 'src': the location of the relevant resource (font, css, images)
+             * - 'prefix': prefix to place before each icon name for convenience
+             *             (ie. fa fa- for FontAwesome, default "")
+             */
+            'FontAwesome' => [
+                'template' => 'font',
+                'prefix' => 'fa fa-',
+                'src' => 'vendor/font-awesome.min.css',
+            ],
+            'Remix' => [
+                'template' => 'svg-sprite',
+                'prefix' => 'ri-',
+                'src' => 'remixicon.symbol.svg',
+            ],
+        ],
```

Comment:
An example (even commented-out) for image-based set would be nice.

Reply:
This is something I'll do over in the application PR (#1962).

Reply:
I see that the Fugue example in bootprint3 in #1962 covers this situation, so I am resolving this thread now.